app/src/main/java/com/example/dine_and_donate/Activities/HomeActivity.java

```diff
@@ -1,51 +1,86 @@
 package com.example.dine_and_donate.Activities;
 
+import android.content.Intent;
 import android.os.Bundle;
 import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
 
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.ActionBar;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.drawerlayout.widget.DrawerLayout;
 import androidx.fragment.app.Fragment;
 import androidx.fragment.app.FragmentManager;
 
+import com.example.dine_and_donate.EditProfileActivity;
 import com.example.dine_and_donate.HomeFragments.MapFragment;
 import com.example.dine_and_donate.HomeFragments.NotificationsFragment;
 import com.example.dine_and_donate.HomeFragments.ProfileFragment;
+import com.example.dine_and_donate.LoginActivity;
+import com.example.dine_and_donate.Models.User;
 import com.example.dine_and_donate.R;
+import com.example.dine_and_donate.ShareEventActivity;
 import com.google.android.material.bottomnavigation.BottomNavigationView;
+import com.google.android.material.navigation.NavigationView;
+
+import org.parceler.Parcels;
 
 public class HomeActivity extends AppCompatActivity {
 
     private BottomNavigationView bottomNavigationView;
+    private FragmentManager fragmentManager;
```

Comment:
doesn't seem like you need this field. you can just do getSupportFragmentManager() in like 48

Reply:
line**